.github/actions/check-pr-status/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "check-pr-status",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "NODE_ENV=production ncc build index.js -o dist --minify"
```

Comment:
Just wondering why is this needed? Can't just pass our `index` to the action? Is it just to beneficiate from the minifier?

Reply:
Actually it's because github actions don't install the dependencies so I have to build a dist file that contains the deps and commit it

Reply:
Alright, that's nice to know, maybe we could put that in the readme? (like, how to update/contribute?)